docs/languages/javascript.md

```diff
@@ -348,4 +348,6 @@ With `javascript.validate.enable: false`, you disable all built-in syntax checki
 
 ### Can I use other JavaScript tools like Flow?
 
-Yes, but some of [Flow's](https://flow.org) language features such as type and error checking may interfere with VS Code's built-in JavaScript support. To learn how to disable VS Code's built-in JavaScript support, see [Disable JavaScript support](/docs/nodejs/working-with-javascript.md#disable-javascript-support).
\ No newline at end of file
+Yes, but some of [Flow's](https://flow.org) language features such as type and error checking may interfere with VS Code's built-in JavaScript support. To learn how to disable VS Code's built-in JavaScript support, see [Disable JavaScript support](/docs/nodejs/working-with-javascript.md#disable-javascript-support).
+
+[^1]: Because of a [bug in TypeScript](https://github.com/Microsoft/TypeScript/issues/6209), _Go to definition_ and _Peek definition_ don’t work if TypeScript typings exist.
```

Comment:
```suggestion
[^1]: Because of a [bug in TypeScript](https://github.com/Microsoft/TypeScript/issues/6209), _Go to definition_ and _Peek definition_ will not work if TypeScript typings exist.
```